sbomdiff/spdx_parser.py

```diff
@@ -34,16 +34,31 @@ def parse_spdx_tag(self, sbom_file):
         packages = {}
         package = ""
         version = None
+        githubStr = "pkg.go.dev/"
         for line in lines:
             line_elements = line.split(":")
             if line_elements[0] == "PackageName":
+                isProductNameWithOnlyVersionNumber = False
                 package = line_elements[1].strip().rstrip("\n")
+                productNameWithOnlyVersionNumber = re.compile(r'(/)')
+                if bool(productNameWithOnlyVersionNumber.search(package)) != True:
+                    isProductNameWithOnlyVersionNumber = True
                 version = None
                 license = None
             if line_elements[0] == "PackageVersion":
                 version = line[16:].strip().rstrip("\n")
             if line_elements[0] == "PackageLicenseConcluded":
                 license = line_elements[1].strip().rstrip("\n")
+            if line_elements[0] == "PackageHomePage":
```

Comment:
If we add this check,we also need to do a corresponding check for all formats of SBOMs not just SPDX tag value SBOMs.

Reply:
Yes I will add same check for json, xml, yaml and rdf. 
Thanks @anthonyharrison